### PR TITLE
feat(autoscaling): reduce the min size of the default machine pool to 3

### PR DESCRIPTION
**Pull request by machi1990**
## Description

Reduce the min size of the default machine pool to 3. This goes in line with https://issues.redhat.com/browse/MGDSTRM-9519 and we'd like to let the autoscaler scale the machine pool from the minimum number of nodes as possible, and 3 seemed
reasonable enough.

## Verification Steps

Tests to continue passing

## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA have been completed
- [ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)
- [ ] Documentation added for the feature
- [ ] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer
- [ ] All PR comments are resolved either by addressing them or creating follow up tasks
- [ ] Required metrics/dashboards/alerts have been added (or PR created).
- [ ] Required Standard Operating Procedure (SOP) is added.
- [ ] JIRA has been created for changes required on the client side

### CODE REVIEW

**Review by pawelpaszki — APPROVED** [review]
lgtm

File: internal/kafka/internal/clusters/cluster_builder.go [hunk 1]
```diff
@@ -87,7 +87,7 @@ func (r clusterBuilder) NewOCMClusterFromCluster(clusterRequest *types.ClusterRe
 	// Set compute node size
 	clusterBuilder.Nodes(clustersmgmtv1.NewClusterNodes().
 		ComputeMachineType(clustersmgmtv1.NewMachineType().ID(computeMachineType)).
-		AutoscaleCompute(clustersmgmtv1.NewMachinePoolAutoscaling().MinReplicas(6).MaxReplicas(18)))
+		AutoscaleCompute(clustersmgmtv1.NewMachinePoolAutoscaling().MinReplicas(3).MaxReplicas(18)))
```
Comment (JameelB) [inline internal/kafka/internal/clusters/cluster_builder.go:90]: Can we define this as a constant? Will this be the same configuration for gcp too?

Comment (machi1990) [reply internal/kafka/internal/clusters/cluster_builder.go:90]: > Can we define this as a constant? 

Good idea. I'll do so.

> Will this be the same configuration for gcp too?

Good question. There hasn't been any indication to change this (especially the maximum) for gcp.

Comment (machi1990) [reply internal/kafka/internal/clusters/cluster_builder.go:90]: raised the configuration question for gcp internally.

**Review by JameelB — APPROVED** [review]

**Pull request merged**